src/SecureElementReader.App/Views/CertDetailsView.axaml

```diff
@@ -95,17 +99,19 @@
 		 </Grid>
 	
      
-    <Rectangle HorizontalAlignment="Stretch" Fill="#1b7dbc" Height="3" Margin="5" />
+    <Rectangle HorizontalAlignment="Stretch" Fill="Gray" Height="3" Margin="5" />
 
      <StackPanel Orientation="Vertical" Width="780" HorizontalAlignment="Left">
-       <Label Content="{Binding SeVerify}" Foreground="{Binding SeColor}" FontSize="15" FontWeight="Bold" Padding="0" />
-       <Label Content="{Binding PkiVerify}" Foreground="{Binding PkiColor}" FontSize="15" FontWeight="Bold" Padding="0"/>
-       <Label FontSize="15" FontWeight="Bold" Padding="0" Content="{Binding SubmitInternalDataStatus}"/>
-       <Label FontSize="15" FontWeight="Bold" Padding="0" Content="{Binding PendingCommandsStatus}"/>
+       <Label Content="{DynamicResource SeCertValid}" IsVisible="{Binding SeCertValid}" Foreground="Green" FontSize="15" FontWeight="Bold" Padding="0" />
+       <Label Content="{DynamicResource SeCertInvalid}" IsVisible="{Binding SeCertInvalid}" Foreground="Red" FontSize="15" FontWeight="Bold" Padding="0"/>
+       <Label Content="{DynamicResource PkiCertValid}" IsVisible="{Binding PkiCertValid}" Foreground="Green" FontSize="15" FontWeight="Bold" Padding="0" />
+       <Label Content="{DynamicResource PkiCertInvalid}" IsVisible="{Binding PkiCertInvalid}" Foreground="Red" FontSize="15" FontWeight="Bold" Padding="0"/>
+       <Label Name="lblCommnads" FontSize="15" FontWeight="Bold" Padding="0" />
+       <Label Name="lblAudit" FontSize="15" FontWeight="Bold" Padding="0" />
        <Button Command="{Binding VerificationInfoCommand}"
```

Comment:
Move this button on top of this region 
![image](https://user-images.githubusercontent.com/68274836/199224809-b2eba0ed-f284-4941-adce-bcddb9591f4f.png)